frontend/src/components/Root/components/Menu/components/ProjectSettingsMenuContent/ProjectSettingsMenuContent.tsx

```diff
@@ -31,7 +31,7 @@ export const ProjectSettingsMenuContent: React.FunctionComponent<Props> = ({
 }) => {
 
   const settingPages = [
-    routeDefinitions.projectSettingsGeneral, routeDefinitions.environmentSettings
+    routeDefinitions.projectSettingsGeneral, routeDefinitions.environmentSettings, routeDefinitions.scriptsAndPagesSettings
```

Comment:
Could you change to `pagesAndScripts` (even in the tab title)? Pages are 70%+ of all the audits on Falco, it makes sense to put it first 👍